recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
+{% set occt_ver = "7.5.3" %}
+{% set ocp_tweak = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # currently only used for the license file
+  - url: https://raw.githubusercontent.com/CadQuery/OCP/{{ occt_ver }}.{{ ocp_tweak }}/LICENSE
+    fn: {{ name }}-{{ version }}-LICENSE.txt
+    sha256: a13caea71627202ad33cc4cafafdd18e667e16716488f8d9c568127121fb89fd
+
+  # binding sources
```

Comment:
What's the reason for having per-platform sources?

Reply:
The sources are generated and have platform specific parts